src/manager.js

```diff
@@ -252,6 +252,7 @@ AdManager.prototype.onSlotRenderEnded = function (event) {
 
   if (event.isEmpty) {
     element.setAttribute('data-ad-load-state', 'empty');
+    element.style.display = 'none';
```

Comment:
@dannypcheng, we should handle this through css.  You should be able to target using this selector `.dfp[data-ad-load-state~=empty]` or something to that effect, which would mean this repo shouldn't be where this type of change lives.

Reply:
OK, sounds good. I'll delete the PR and make the css change in mantle.